agent/wsclient/client_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const dockerEndpoint = "/var/run/docker.sock"
```

Comment:
This shouldn't be a constant.  The default is `/var/run/docker.sock` on Linux and `\\.\npipe\docker_engine` on Windows, but in both cases it can be overridden.  Can you read `cs.AgentConfig.DockerEndpoint` instead?  (See my previous comment: https://github.com/aws/amazon-ecs-agent/pull/659#discussion_r95906016